connector/interface.go

```diff
@@ -60,6 +60,11 @@ type ConnectorConfig interface {
 	Connector(ns url.URL, loginFunc oidc.LoginFunc, tpls *template.Template) (Connector, error)
 }
 
+// Groups is an optional interface implemented by a Connector.
+type Groups interface {
```

Comment:
Maybe: "GroupsConnector"? Idk, derivatives of "Directory" or "Membership" seemed a little odd or specific.